### DISCUSSION TITLE
Will playing DASH and/or HLS be supported?

### DISCUSSION THREAD

**Discussion by PeterVanVogelpoel** [CommunityToolkit/Maui#1154] [New Feature Discussions]
I know it's mentioned here and there, but I don't find a clear answer.

Will playing DASH and/or HLS with the use of an Autorization token (through a header) be supported?

Would be much appreciated.
Peter

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
